### ISSUE TITLE
Material You icon support for Aegis

### ISSUE THREAD

**Issue by Techkkq** [beemdevelopment/Aegis#939]
Hello, 
First of all I just wanted to say that I love your app guys and
I would like if you would add Material You icon support for Aegis.
https://developer.android.com/about/versions/13/features#themed-app-icons

Thank you

**Comment by alexbakker** [comment]
Agreed. This should be fairly easy to do.

**Comment by ChanceHarrison** [comment]
Potentially related to https://github.com/beemdevelopment/Aegis/issues/842?

**Comment by alexbakker** [comment]
Indeed. That one doesn't require a targetSdkVersion bump though, but I believe this one does (which is currently blocked by https://github.com/android/android-test/issues/1412).

**Comment by pwd-github** [comment]
the blocking bug is fixed! 🙌

**Comment by jas14** [comment]
Hi all 👋 I just wrote a branch to do this, but when I bumped the target SDK to 33, [robolectric](https://github.com/robolectric/robolectric) tests started failing. Looks like it doesn't yet support API level 33.

I'm happy to open the PR with Robolectric configured to run against API level 32, but that seems like it might defeat the purpose of tests a bit... Let me know. In the meantime, here's a sneak peek:

<img src="https://user-images.githubusercontent.com/569991/188295812-14fa0962-be57-4eb9-b822-f7bc5e66b4fc.png" width="300"><img src="https://user-images.githubusercontent.com/569991/188295832-f0540e6e-85bc-4574-88c4-f1968a6e2fec.png" width="300">

**Comment by alexbakker** [comment]
@jas14 Nice!

I'm not too worried about forcing the Roboelectric tests to API 32. We can switch that back later, after Roboelectric adds support for API 33.

**Comment by pwd-github** [comment]
Looks amazing @jas14

**Comment by pwd-github** [comment]
This was fixed a while back. Any insight into what needs to happen to get a version released that supports this @alexbakker ?

**Comment by pwd-github** [comment]
> @pwd-github There's now a beta release that contains this change! [v2.1-beta1](https://github.com/beemdevelopment/Aegis/releases/tag/v2.1-beta1)

You're a lovely, wonderful man. Thank you so much for your continued hard work.
